src/time.rs

```diff
@@ -253,6 +253,30 @@ impl IntoValue for SystemTime {
     }
 }
 
+#[cfg(feature = "chrono")]
```

Comment:
If you add `#[cfg_attr(docsrs, doc(cfg(feature = "chrono")))]` after each use of `#[cfg(feature = "chrono")]`, like:

``` rust
#[cfg(feature = "chrono")]
#[cfg_attr(docsrs, doc(cfg(feature = "chrono")))]
impl ...
```

then rustdoc (well, at least with with config that docs.rs uses, run `env RUSTDOCFLAGS='--cfg docsrs' cargo +nightly doc --all-features` to see it locally) will show a little "Available on crate feature chrono only." flag next to this item in the docs.